en/docs/includes/deploy/assign-custom-hostname.md

```diff
@@ -18,13 +18,13 @@
 
 3.  Click on the API to edit its configurations.
 
-     [![Edit the API](/../../../../assets/img/includes/deploy/select-api.png)](/../../../../assets/img/includes/deploy/select-api.png)
+     [![Edit the API](../../assets/img/includes/deploy/select-api.png)](../../assets/img/includes/deploy/select-api.png)
 
 4.  Click **Deployments**.
 
 5.  Select the newly created Gateway environment, Virtual Host (foods.com), and a Revision.
 
-     [Create a new revision](/../../../../design/create-api/create-api-revisions) if no revisions exist.
+     [Create a new revision](../../design/create-api/create-api-revisions.md) if no revisions exist.

```

Comment:
Use base path notation.

`[Create a new revision]({{base_path}}/design/create-api/create-api-revisions/)`

Reply:
{{base_path}} did not worked in included when we tests @Mariangela